_inc/client/main.jsx

```diff
@@ -118,8 +118,9 @@ class Main extends React.Component {
 
 	initializeAnalyitics = () => {
 		const tracksUser = this.props.tracksUserData;
+
 		if ( tracksUser ) {
-			analytics.initialize( tracksUser.userid, tracksUser.username );
+			analytics.initialize( tracksUser.userid, tracksUser.username, { blogid: tracksUser.blogid } );
```

Comment:
I'm a bit concerned that we're initializing analytics in several locations, but setting `blogid` only in this one. Should we make sure that any initialization of analytics throughout Jetpack can benefit from this change?

Reply:
This is the only instance that uses that client-side library. The other with similar names are PHP-side events.

Reply:
@AnnaMag I am talking about these instances, which don't seem to be addressed in this PR:

https://github.com/Automattic/jetpack/blob/4e716c7bdf6aae6dd5fe39c9029a509c7c2bee6f/_inc/idc-notice.js#L21

https://github.com/Automattic/jetpack/blob/f8078c2cd12ac508334da2fb08e37a92cf283c14/modules/widgets/search/js/search-widget-admin.js#L16

Thoughts?

Reply:
they are handled by another tracks emitting module on the servers-side. This is not part of this PR.